app/models/manageiq/providers/lenovo/manager_mixin.rb

```diff
@@ -73,7 +83,15 @@ def discover_queue(username, password, zone = nil)
     private
 
     def discover_from_queue(username, password)
-      discover(username, MiqPassword.decrypt(password))
+      discover(username, '')
```

Comment:
Something's incongruous here :smile: 
Passing `username` and `''` to a method expecting `ip_address` and `port`.

Seems like the caller of `discover_queue` should be passing all of `username`, `password`, `ip_address`, and `port`.  Then, all of that should be queued up for this method, and all of those params passed to the `discover` method.